packages/object-values/README.md

```diff
@@ -13,7 +13,7 @@ values({a: {aa: 2}, b: {bb: 4}}); // [{aa: 2}, {bb: 4}]
 values({}); // []
 values([1, 2, 3]); // [1, 2, 3]
 values(function(a, b) {return a + b;}); // []
-values(new String('hello')); // ['h', 'e', 'l', 'l', 'o']
+values(new String('hello')); // throw exception
```

Comment:
This shouldn't throw an exception and doesn't with current code and your TS definition. (It's an object, your TS test omits the `new` qualifier so makes it a string). Also my fault because I forgot the `new` in `index.js` comment :)